### ISSUE TITLE
[ Suggestion ] CURIOS SUPPORT

### ISSUE THREAD

**Issue by Camawama** [McJty/TheOneProbe#574]
I looked through the issues for a duplicate, maybe I missed one.

It would be amazing if you added Curios Support. It's just as simple as a TAG.

**Comment by a-frenzy** [comment]
Yes please! https://github.com/TheIllusiveC4/Curios/wiki/How-to-Use:-Developers
